apps/server/lib/lexical/server/code_intelligence/completion.ex

```diff
@@ -2,6 +2,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion do
   alias Lexical.Completion.Translatable
   alias Lexical.Project
   alias Lexical.Protocol.Types.Completion
+  alias Lexical.Protocol.Types.Completion.List, as: CompletionList
```

Comment:
slight preference for `Completion.List`, and just alias `Types.Completion`

Reply:
Because in Elixir, the `List` gives the impression of `[]`, and `List.new` feels strange. Although, I know that in JavaScript all lists are objects.

Reply:
you would do `Completion.LIst.new`, not `List.new`